uploads/upload.ts

```diff
@@ -0,0 +1,92 @@
+const upload = require("express").Router();
+const express = require("express");
+const dotenv = require("dotenv");
+import { Project } from "../models/Model";
+
+dotenv.config();
+
```

Comment:
I believe these imports need refactored to typescript imports. I think this is causing the start script to break for the built application. 

If you run the build script then try to start the app like it would when it deploys it throws an error for the multer import